Automattic/jetpack
# Fix issue with Donation block in Danish Krone

Fixes https://github.com/Automattic/wp-calypso/issues/81006

## Proposed changes:

* Fix issue with Danish Krone as it had no separator

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to a new post
* Add a donation block
* Make sure switching to Danish did not break the block
